OpenRA.Game/FileSystem/FileSystem.cs

```diff
@@ -322,6 +322,46 @@ public static string ResolveAssemblyPath(string path, Manifest manifest, Install
 			return File.Exists(resolvedPath) ? resolvedPath : null;
 		}
 
+		public static string ResolveCaseInsensitivePath(string path)
+		{
+			var remaining = path.Split("/");
```

Comment:
`var remaining = path.Split(Path.DirectorySeparatorChar, Path.AltDirectorySeparatorChar);` to also cover the Windows backslash case?

Reply:
My decompiler tells me, that both variables are '/'... Ill take the string variant then

Reply:
The values are platform-specific, see https://learn.microsoft.com/en-us/dotnet/api/system.io.path.altdirectoryseparatorchar?view=net-6.0

Reply:
Yeah, but if one would use the \ char in its installer.yaml, my linux wont resolve that correctly as both variables are /

Reply:
We define/require/assume that all OpenRA/yaml paths use /, so that won’t happen in practice